### PR TITLE
stars

### PR DESCRIPTION
**Pull request by ISRLv2021**
- [DEMO LINK](https://ISRLv2021.github.io/layout_stars/)
- [TEST REPORT LINK](https://ISRLv2021.github.io/layout_stars/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,30 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  margin: 0;
```
Comment (Anna-Tkachenko) [inline src/style.css:7]: This block has margin: 0 by default, remove this styles

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  margin-left: 0;
+}
+
+.stars--1 .star:nth-child(1),
+.stars--2 .star:nth-child(-n+2),
+.stars--3 .star:nth-child(-n+3),
+.stars--4 .star:nth-child(-n+4),
+.stars--5 .star:nth-child(-n+5) {
+  background: url(./images/star-active.svg) no-repeat;
+  background-position: center;
```
Comment (Anna-Tkachenko) [inline src/style.css:29]: Move background-position and background-repeat properties to the main star class
You should change only background-image for active stars

**Review by Anna-Tkachenko — CHANGES_REQUESTED** [review]

File: src/style.css (hunk 1 above)
Comment (ISRLv2021) [reply src/style.css:7]: Right! Thank you

**Review by Anna-Tkachenko — APPROVED** [review]
Good job!

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+}
+
+.stars {
+  display: flex;
+}
+
+.star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
```
[the remark this answers is not in the document]
Comment (ISRLv2021) [reply src/style.css:12]: .star:not(nth:last-child) { 
  margin-right: 4px;
}
would this solution work?

**Pull request closed without merging**